lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -282,16 +282,20 @@ def unlink(self, missing_ok: bool = False):
         source_tarball, source_md5, md5 = tarball
         monkeypatch.setattr(Tarball, "_get_metadata", fake_get_metadata)
         cm = CacheManager(server_config, make_logger)
-        real_move = shutil.move
         monkeypatch.setattr("pbench.server.cache_manager.shutil.move", mymove)
         monkeypatch.setattr(Path, "unlink", unlink)
         with pytest.raises(Exception) as e:
             cm.create(source_tarball)
         assert str(e.value) == "I'm broken"
         assert src == [source_md5] + ([source_tarball] if allow else [])
-        for i in range(len(ulink)):
-            assert ulink[i] == dest[i]
-        assert ok
+        assert len(src) == len(dest) == len(ulink) == len(ok) == 2 if allow else 1
+        for i, s in enumerate(src):
+            assert dest[i].name == s.name
+            assert ulink[i].name == s.name
+            assert dest[i] == ulink[i]
+        assert all(
+            o for o in ok
+        ), f"All cleanup unlinks should ignore missing: {ok}, {ulink}"
```

Comment:
I don't think you need the explicit iteration -- I think `all()` will accept any _iterable_, and I think `ok` is an iterable, in which case `all()` will test each element in `ok`...so adding `o` doesn't actually add anything, except a prompt to Black to wrap the line....  🦐

```python
        assert all(ok), f"All cleanup unlinks should ignore missing: {ok}, {ulink}"
```

Reply:
Ha. You're not wrong; although to get black to pull it back onto one line I have to shorten the message, too.